src/index.js

```diff
@@ -4,11 +4,34 @@ import './index.css';
 import App from './App/App';
 import { BrowserRouter } from 'react-router-dom';
 import reportWebVitals from './reportWebVitals';
+import {
+  ApolloProvider,
+  ApolloClient,
+  createHttpLink,
+  InMemoryCache
+} from '@apollo/client';
+
+// const httpLink = createHttpLink({
+//   uri: 'https://graphqlzero.almansi.me/api'
+// });
+
+const currURI = (process.env.NODE_ENV === 'development') ? '/graphql' : 'https://aqueous-savannah-80171.herokuapp.com/graphql';
```

Comment:
This whole chunk is what is still a mystery to me. One of those "I guess this is just how you do it, because that's how it's designed to be done" things. A little gnarly though. I wish there was better documentation on this.